readme.md

```diff
@@ -1,4 +1,4 @@
-# google-proto-files [![Build Status][build_img]][build_link]
+# nodejs-proto-files [![Build Status][build_img]][build_link]
```

Comment:
I think the name can stay. One reason would be it's the name of the module on `npm`. Another, these aren't "node" proto files, but very specific to Google and its APIs.

Reply:
Fixed.